main/lsp/BUILD

```diff
@@ -25,7 +26,7 @@ cc_library(
         "lsp_messages_gen_helpers.h",
         "watchman/WatchmanProcess.h",
         "wrapper.h",
-    ],
+    ] + glob(["requests/*.h"]),
```

Comment:
ooc, is there a reason why not to glob all header files?

Reply:
Good question.

We should probably just do that until we decide to prune the header list. (These are listed here separately because they are exported from the package, right...?)

Reply:
Just pushed a commit that significantly prunes down `hdrs` to just the interfaces consumed externally or in tests. It could be pruned even further with a little bit of work.